.github/actions/setup/action.yml

```diff
@@ -7,30 +7,24 @@ description: Sets this repository up to allow efficient run of various github ac
 runs:
   using: composite
   steps:
+    - uses: pnpm/action-setup@v2
+      name: Install pnpm
+      id: pnpm-install
+      with:
+        version: 8
+        run_install: false
     # setup node
     - uses: actions/setup-node@v3
       with:
         node-version-file: .nvmrc
-        cache: "yarn"
-        registry-url: https://npm.pkg.github.com/
+        registry-url: https://npm.pkg.github.com
+        cache: "pnpm"
+        node-version: 18
```

Comment:
Aren't these two doing the same thing? 🤔

Reply:
In what way? The first Line 10 installs the `pnpm` binary so that we can use the command in other scripts. Line 17+ sets up node.